pypeit/data/arc_lines/lists/ArII_lines.dat

```diff
@@ -3,33 +3,110 @@
+| ArII |  5018.5619 |    1 |     0 |     10000 |                       NIST |
 | ArII |  5063.4483 |    1 |    64 |        47 |  GMOS_R400_blue_1_fit.json |
+| ArII |  5091.9136 |    1 |     0 |     10000 |                       NIST |
+| ArII |  5127.1945 |    1 |     0 |     10000 |                       NIST |
+| ArII |  5143.2151 |    1 |     0 |     10000 |                       NIST |
+| ArII |  5146.7414 |    1 |     0 |     10000 |                       NIST |
+| ArII |  5441.5011 |    1 |     0 |     10000 |                       NIST |
 | ArII |  5455.1500 |    1 |    64 |        27 | GMOS_R400_blue_15_fit.json |
+| ArII |  5579.2345 |    1 |     0 |     10000 |                       NIST |
+| ArII |  5693.2422 |    1 |     0 |     10000 |                       NIST |
+| ArII |  5788.1660 |    1 |     0 |     10000 |                       NIST |
+| ArII |  5814.3720 |    1 |     0 |     10000 |                       NIST |
+| ArII |  6048.5721 |    1 |     0 |     10000 |                       NIST |
 | ArII |  6105.2293 |    1 |    64 |        33 | GMOS_R400_blue_38_fit.json |
 | ArII |  6116.6158 |    1 |    64 |        62 | GMOS_R400_blue_38_fit.json |
+| ArII |  6125.0558 |    1 |     0 |     10000 |                       NIST |
 | ArII |  6173.9855 |    1 |    64 |        99 | GMOS_R400_blue_38_fit.json |
-| ArII |  6484.87395|    1 |    64 |        44 |  GMOS_B600 |
+| ArII |  6202.8159 |    1 |     0 |     10000 |                       NIST |
+| ArII |  6326.1639 |    1 |     0 |     10000 |                       NIST |
+| ArII |  6439.3805 |    1 |     0 |     10000 |                       NIST |
+| ArII |  6484.87395|    1 |    64 |        44 |                  GMOS_B600 |
+| ArII |  6616.1760 |    1 |     0 |     10000 |                       NIST |
+| ArII |  6622.7940 |    1 |     0 |     10000 |                       NIST |
+| ArII |  6640.0534 |    1 |     0 |     10000 |                       NIST |
+| ArII |  6641.5739 |    1 |     0 |     10000 |                       NIST |
+| ArII |  6645.5319 |    1 |     0 |     10000 |                       NIST |
+| ArII |  6668.1994 |    1 |     0 |     10000 |                       NIST |
+| ArII |  6686.1380 |    1 |     0 |     10000 |                       NIST |
+| ArII |  6863.1621 |    1 |     0 |     10000 |                       NIST |
+| ArII |  7591.4046 |    1 |     0 |     10000 |                       NIST |
 | ArII |  8330.1960 |    1 |    64 |       319 | GMOS_R400_chip2_8_fit.json |
 | ArII |  8365.3720 |    1 |    64 |       391 | GMOS_R400_chip2_8_fit.json |
 | ArII |  8774.2691 |    1 |    64 |      8086 |      GMOS_R830_shenli.json |
+| ArII |  9010.9298 |    1 |     0 |     10000 |                       NIST |
+| ArII |  9020.0670 |    1 |     0 |     10000 |                       NIST |
 | ArII |  9438.819  |    1 |    64 |      8086 |      GMOS_R830_shenli.json |
+| ArII |  9511.0480 |    1 |     0 |     10000 |                       NIST |
```

Comment:
What is determining the red cutoff in this line list? We frequently have the problem that nobody compiles lines at > 1 um, and that means when we have red setups, someone has to go back and try to figure out how to get the lines again to get decent fits, whereas the person who originally made the list could save everyone a ton of time if they just added the redder lines on the first pass.

Reply:
The lines I added are from the IRAF line list `fear.dat`, that I cross-matched with NIST line lists to get vacuum wavelength and confirm/correct ion IDs. If you want I add lines > 1 µm, maybe cutting on something with more physical meaning than relative intensity (which is how some of the prominent IRAF lines were lost), but I would prefer to do that in a different PR.


 fear.dat -  FeAr Hollow Cathode line list between 3,000A and 10,000A.
		   The reference for this line list is "A CCD Atlas of
		   Comparison Spectra: FeAr Hollow Cathode, 3000A - 10,000A"
		   by D. Willmarth.  The atlas is available from the
		   KPNO User Support Office.

see e.g. https://github.com/iraf-community/iraf/tree/main/noao/lib/linelists

Reply:
I don't understand. There are still lines like 

9508.4513  ArI

that show up in the IRAF fear.dat list but are not in our ArI line list. Why is this line excluded? Is it because it is off your wavelength coverage for P200? Is it absent in your spectrum, and is that because it a weak line according to NIST?

To be clear, I'm not asking you to cover the entire near-IR which would be a lot of work. I'm just asking that for the IRAF line lists that you did consult (fear.dat) please make sure that all lines available in those lists have been cross matched with NIST up to the red coverage cutoff of those lists (not the red coverage cutoff of your data).

Reply:
That line has vaccuum wavelength 9511.0480, and NIST lists it as an ArII line.
Manually checking the IRAF lines, I found a fair number that were initially excluded because NIST gave a Ritz wavelength for them and no observed wavelength. I just added all of these lines back in.

There were several the IRAF lines that I was not able to cross-match with NIST lines (e.g. 8143.5050 air ArI, 5514.3760 ArI), and those are still excluded.